### ISSUE TITLE
Archival Collection

### ISSUE THREAD

**Issue by erazlogo** [citation-style-language/zotero-bits#27]
Archival Collection type had been discussed on the forum and tickets, and will definitely need to be added for hierarchical item types, but it would be nice to add them now. Online archival collections are similar to Data Sets, but physical collections are different obviously.

https://github.com/ajlyon/zotero-bits/wiki/ArchivalCollectionType

**Comment by bdarcus** [comment]
FWIW, when I designed CSL originally, I was writing a manuscript that included reference to many archival documents. But I never considered the possibility to cite the collections themselves. I was thinking of collections much like a periodical; a collection of documents you don't cite on their own.

**Comment by alexduryee** [comment]
Adding support for this issue/feature, following discussions with fellow archivists.  My experience from trying to implement Zotero support for http://archives.nypl.org/ has been that the current data models don't really support archival metadata/collections, so this is an exciting feature to see implemented.

There's a draft extension to Schema.org for representing archival repositories and collections (https://archival.github.io/schema-org/about/), which may be a good place to start regarding modeling/mapping archival collections.  From the perspective of implementation, it would allow for Zotero to only need a single translator, which would work on any site supporting the extended standard -- as well as providing a strong use case for implementing Schema.org for archival repositories.

Regarding modeling Archival Collections - would these be modeled as Items or Collections?  The advantage of using a Collection model would be the built-in support for hierarchical data, which would allow for importing full finding aids into Zotero.  This would enable things like folder/item-level citation generation and the ability to attach digital images to granular archival components (ref. https://guides.library.harvard.edu/zotero_archival_research for a sample use case).

**Comment by bwiernik** [comment]
APA style also requires citing the geographic location of the the archive, so `archive-place` also seems like a necessary additional field.

**Comment by adam3smith** [comment]
Yes, of course -- `archive-place` is already in CSL and should be part of however Zotero decides to model archives.

**Comment by bwiernik** [comment]
Ah, forgot archive-place was already in CSL
